todo-list/src/DayList.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { ACCENT_COLOURS, COLOURS } from "./colours";
+
+const DayList = (props) => {
+  const numTodoItems = 10;
+  const { day, isDarkMode, accentColour } = props;
+  const [isChecked, setChecked] = useState(
+    Array.from({ length: numTodoItems }).fill(false)
+  );
+  const [textInputValues, setTextInputValues] = useState(
+    Array.from({ length: numTodoItems }).fill("")
+  );
+
+  const todoItemStyles = {
+    color: isDarkMode ? COLOURS.White : COLOURS.Black,
+    backgroundColor: isDarkMode ? COLOURS.Black : COLOURS.White,
+  };
+
+  const checkBoxStyles = {
+    accentColor: accentColour,
+  };
+
+  const handleCheckboxChange = (index) => {
+    const newCheckedState = [...isChecked];
+    newCheckedState[index] = !newCheckedState[index];
+    setChecked(newCheckedState);
+  };
+
+  const handleTextInputChange = (event, index) => {
+    const { value } = event.target;
+    setTextInputValues((prevValues) => {
+      const updatedValues = [...prevValues];
+      updatedValues[index] = value;
+      return updatedValues;
+    });
+  };
+
+  const handleKeyDown = (event, index) => {
+    if (event.key === "Enter" && textInputValues[index]) {
+      const nextIndex = index + 1;
+      if (nextIndex < numTodoItems) {
+        document.getElementById(`todo-input-${day}-${nextIndex}`).focus();
+      }
```

Comment:
Input was going always going to Monday on enter since it was only considering the row (index) and not the column (day). So I changed `todo-input-${nextIndex}` to `todo-input-${day}-${nextIndex}` and it is fixed.